xrtpy/response/channel.py

```diff
@@ -11,16 +11,14 @@
 ]
 
 import numpy as np
-import pkg_resources
 import scipy.io
 import sunpy.io.special
 import sunpy.time
 
 from astropy import units as u
+from pathlib import Path
 
-filename = pkg_resources.resource_filename(
-    "xrtpy", "response/data/xrt_channels_v0016.genx"
-)
+filename = Path(__file__).parent.absolute() / "data" / "xrt_channels_v0016.genx"
```

Comment:
I ran into some issues on Windows with paths not being recognized, so I ended up using `pathlib` instead which I learned recently is the recommended way to deal with paths in Python nowadays.

Reply:
Awesome, thanks for the note!